app/src/main/kotlin/com/mapzen/erasermap/view/InstructionListActivity.kt

```diff
@@ -60,7 +60,7 @@ public class InstructionListActivity : AppCompatActivity() {
         return true
     }
 
-    private class DirectionListAdapter(context: Context, strings: ArrayList<String>?,
+    public class DirectionListAdapter(context: Context, strings: ArrayList<String>?,
```

Comment:
Maybe we should consider breaking this out into its own file and some point since its used in more than one place?